base/uk.ac.stfc.isis.ibex.opis/resources/RIKEN/individual_kicker_row.opi

```diff
@@ -65,16 +65,11 @@
     <rotation_angle>0.0</rotation_angle>
     <rules>
       <rule name="Rule" prop_id="background_color" out_exp="false">
-        <exp bool_exp="pv0 == 0">
+        <exp bool_exp="pv0 == 2">
```

Comment:
Please could you add any rules like this used to `addFastPathHandlers()` in OPI.java if there not already present or in `RhinoWithFastPathScriptStore` if you need help with this feel free to ask me or Tom.